aosp_xenvm_trout_arm64.mk

```diff
@@ -45,7 +45,7 @@ include device/epam/aosp-xenvm-trout/aosp_xenvm_trout_common.mk
 DEVICE_MANIFEST_FILE := device/google/trout/manifest_x86.xml
 DEVICE_MATRIX_FILE += device/google/trout/compatibility_matrix.xml
 DEVICE_FRAMEWORK_COMPATIBILITY_MATRIX_FILE := device/google/trout/framework_compatibility_matrix.xml
- 
```

Comment:
Comment on patch "Extend rootfs size to 2Gb" : this change not needed.

Reply:
Hi @arminn, it is hard to follow this comment after some work is done ( GitHub screws it up ). But it seems to be removed now.

Done.